ucsd-fair-data-informatics.md

```diff
@@ -0,0 +1,67 @@
+# UCSD Fair Data Informatics Lab - Neuroscience Information Framework
+## Description
+The University of California, San Diego (UCSD) Fair Data and Informatics Laboratory (FDI Lab) supports open science and is 
+home to the Neuroscience Information Framework (NIF). NIF has extensive experience in community
+engagement, collaborative standards building, effective processes for sharing, federating, and
+integrating neuroscience data, information, and knowledge.
+
+# Roles
+[Principal Investigator] Maryann Martone (maryann@ncmir.ucsd.edu)   
+[Co-Investigator] Jeffrey Grethe (jgrethe@ncmir.ucsd.edu)   
+[Co-Investigator] Anita Bandrowski (abandrowski@ncmir.ucsd.edu)  
+[Knowledge Engineer] Tom Gillespie (tom.h.gillespie@gmail.com)   
+
+## Definitions [Optional]
+* __NIFSTD__ Neuroscience Information Framework (NIF) Standard ontology (NIFSTD) is a neuroscience
+  ontology that maintains an extensive set of terms and concepts important for the domains of
+  neuroscience and biology.
+* __InterLex__ A dynamic lexicon of biomedical terms built on the foundation of NeuroLex
+
+## Objectives
+* Develop a semantic framework for annotating and organizing cell level data by maturing community
+  efforts to create a phenotype-based nervous system cell type ontology and phenotype representation
+* Work with the community to develop and implement spatial and semantic standards for managing
+  heterogeneous brain cell census data types
+
+## In-scope
+* Create an effective strategy for semantic modeling and classifying neurons
+* Create a comprehensive ontology that represents canonical types (e.g., Purkinje cell),
+  cell classes (e.g., GABAergic neurons) and individuals (Cell X from laboratory Y) based
+  on neuronal properties and phenotypes.
+* Establish a community platform for consortium members and beyond to easily view the ontology and
+  to recommend and provide feedback on terms in the ontology.
+* Provide the neuron ontology and other NIF ontologies covering additional relevant domains, e.g.,
+  small molecules, subcellular structures, anatomical structures, techniques, via our open-source
+  terminology/ontology service SciGraph
+* Work with the BICCN community to help ensure that infrastructure and tools conform to best
+  practices for making data FAIR: Findable, Accessible, Interoperable and Reusable
+
+## Out-of-scope
```

Comment:
I would move that Out-of-scope is optional and if you do not have any items, this is just not a section that is included.

Reply:
Will remove the out-of-scope section.

Reply:
resolved by 8edb24f